### DISCUSSION TITLE
Is a Scala 3 binary planned?

### DISCUSSION THREAD

**Discussion by kincsescsaba** [coursier/coursier#3033] [Q&A] [answered]
Hi there,

I'd be happy to know what the plans are regarding a Scala 3 release for coursier as I'd like to use it in a Scala 3 environment and I run into issues with cross builds.

**Comment by tgodzik** [comment, marked as answer]
Not currently, since you should be able to just use 2.13 artifacts from Scala 3 without an issue or alternatively the Java API. If there are problems with that let us know

**Reply by kincsescsaba** [reply]
I have an issue that looks like to be rooted in the problem of not adding all the needed jars, I'll try to update my sbt setup to handle this automatically.

**Reply by kincsescsaba** [reply]
An error persists, coursier package is not found (I tried invalidate caches) with this build.sbt setting:
`ThisBuild / version := "0.1.0-SNAPSHOT"

ThisBuild / scalaVersion := "3.3.3"

lazy val supportedScalaVersions = List("3.3.3", "2.13.14")

lazy val coursier = (project in file("coursier"))
  .settings(
    name := "coursier",
    crossScalaVersions := supportedScalaVersions,
    libraryDependencies ++= Seq(
      "io.get-coursier" % "coursier_2.13" % "2.1.10"
    )
  )

lazy val root = (project in file("."))
  .aggregate(coursier)
  .settings(
    name := "class-hotload-poc",
    crossScalaVersions := Nil,
    libraryDependencies ++= Seq(
      //"com.lihaoyi" %% "os-lib" % "0.10.1",
      "org.scala-lang" %% "scala3-compiler" % scalaVersion.value,
      "org.scalatest" %% "scalatest" % "3.2.18" % Test
    ),
    publish / skip := true
  )
`

Any ideas?

**Reply by tgodzik** [reply]
Ach, I think you need to add:
`("io.get-coursier" % "coursier_2.13" % "2.1.10").cross(CrossVersion.for3Use2_13) `

Or use 2.13 version explicitely

**Reply by kincsescsaba** [reply]
Tried this but still had issues with SBT's dependency resolution. I'll try adding all jars as unmanaged binaries

**Reply by tgodzik** [reply]
What kind of problems? Maybe I can help if you show me the error messages.
